docs/source/tutorials/processors.md

```diff
@@ -0,0 +1,70 @@
+# Tutorial: Adding a processor
+
+Processors can be thought of as torchvision transforms which transform a sample into a form usable by the model. Each processor takes in a dictionary and returns back a dictionary. Processors are initialized as member variables of the dataset and can be used while generating samples.
+
+
+For this tutorial, we will create a processor to get sentence embeddings.
+
+## Create the processor class
```

Comment:
Can we add two more examples here?
- A processor from scratch (inheriting the base processor). An example can be around transformer tokenizer.
- A processor using torchvision transforms

Reply:
I will add two more simple processors, one for text and one for image extending `BaseProcessor`